dds_web/database/models.py

```diff
@@ -355,6 +355,8 @@ def __init__(self, **kwargs):
         super(User, self).__init__(**kwargs)
         if not self.hotp_secret:
             self.hotp_secret = os.urandom(20)
+        if not self.kd_salt:
+            self.kd_salt = os.urandom(32)
```

Comment:
I recommend removing lines 358-359, because a salt is paired with a password and there are places where a password is set or changed, and those places are where we set the salt. It is not something that exists on its own or for a lifetime.

Reply:
Ok removed this and put the salt generation in diffrent places in the code. We can come up with a better solution as we go